docs/reference/api-v1.4.rst

```diff
@@ -117,6 +118,40 @@ Example Response:
     DEIS_API_VERSION: 1.4
     DEIS_PLATFORM_VERSION: 1.6.1
 
+Regenerate Token
+````````````````
+
+.. note::
+
+    This command could require administrative privileges
+
+Example Request:
+
+.. code-block:: console
+
+    DELETE /v1/auth/tokens/ HTTP/1.1
```

Comment:
Hmm... Maybe we should make this a POST instead of a DELETE. We're not actually deleting a token here, just generating a new one.